File: cpp_solutions/268. Missing Number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        
        int sum = 0;
        int n = nums.size() ;
        for(int x:nums){
            sum+=x;
        }
        
        return n*(n+1)/2 - sum;
        
    }
};

File: python3_solution_set0/268. Missing Number.py
##ss
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        
        
        
        ans = [-1 for x in range(len(nums)+1)]
        print(ans)
        
        for x in range(len(nums)):
            ans[nums[x]] = nums[x]
            
        return ans.index(-1)
        
